lib/simulation/openfisca/mapping/foyerFiscal.js

```diff
@@ -11,9 +12,10 @@ module.exports = {
     },
     personnes_a_charge: {
         fn: function(situation) {
-            var enfants = _.pluck(_.filter(situation.individus, { role: 'enfant' }), '_id');
-            var personnesACharge = _.pluck(_.filter(situation.individus, { role: 'personneACharge' }), '_id');
-            return enfants.concat(personnesACharge);
+            var enfants = _.filter(situation.individus, function(individu) {
+                return mapping.isIndividuValid(individu, situation) && individu.role == 'enfant';
```

Comment:
I don't get it. This doesn't reads as equivalent to the previous code: we're asking for `personne_a_charge` and returning only children…?